app/presenters/statistics_presenter.rb

```diff
@@ -31,7 +31,7 @@ def version
   end
 
   def open_registrations?
-    AppConfig.settings.enable_registrations
+    AppConfig.settings.enable_registrations?
```

Comment:
@jhass : Nope, [see by yourself](https://travis-ci.org/diaspora/diaspora/jobs/48826537#L447), it doesn't work at all, plus it makes 2 other tests to fail.